app/assets/stylesheets/application.css

```diff
@@ -15,3 +15,29 @@
  *= require foundation_and_overrides
 
  */
+
+ body {
+   background-image: url("background1.jpg");
```

Comment:
Rails provides a path helper you should use here: `image_url("...")` instead of `url(...)`. Change that, and change this file to a `.scss` file!